style.css

```diff
@@ -0,0 +1,227 @@
+.container {
+  background-color: #2b2b2b;
+}
+
+.header {
+  display: flex;
+  flex-wrap: wrap;
+  padding-top: 3%;
+  justify-content: space-evenly;
+}
+
+.lotus {
+  border-radius: 50%;
+}
+
+.topnav {
+  background-color: black;
+  border-radius: 25px;
+
+}
+
+.topnav a {
+  float: left;
+  color: #F3F2F2;
+  text-align: center;
+  padding: 25px 50px;
+  text-decoration: none;
+  font-size: 17px;
+  font-weight: bold;
+}
+
+.topnav a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+/* Add a color to the active/current link */
+.topnav a.active {
+  color: lemonchiffon;
+}
+
+.round {
+  width: 16rem;
+  border-radius: 32px;
+  height: 58px;
+  border: 1px #000 solid;
+  padding: 5px 5px 5px 25px;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 5;
+}
+
+
+.search {
+  position: relative;
+  width: 190px;
+  height: 30px;
+}
+
+.corner {
+  position: absolute;
+  top: 27px;
+  left: 7px;
+  height: 20px;
+  width: 20px;
+  z-index: 10;
+  border-radius: 10px;
+  border: none;
+  color: #C1C0C0;
+}
+
+input, select, textarea {
+  background-color: #464444;
+}
+
+h1 {
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+}
+h4{
+  font-family:sans-serif;
+  color: white;
+  margin-bottom: -13px;
+
+}
+.even-div {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
+
+.top-right {
+  position: absolute;
+  top: 20%;
+  left: 43%;
+}
+
+.heading-align {
+  display: flex;
+  margin-left: 50px;
+
+
+}
+
+p {
+  color: grey;
+  font-family: 'Lucida Sans';
+}
+
+
+.bottom-border:before {
+  content : "";
+  position: absolute;
+  left    : 0;
+  bottom  : 0;
+  height  : 1px;
+  width: 50%; /* or 100px */
+  border-bottom: 2px solid #C8C81F;
+}
+
+/* Bottom left text */
+.bottom-left {
+  position: absolute;
+  bottom: 33%;
+  left: 7%;
+}
+
+.bottom-left2 {
+  position: absolute;
+  bottom: 28%;
+  left: 7%;
+}
+/* Bottom right text */
+.bottom-right {
+  position: absolute;
+  bottom: 32%;
+  right: 41%;
+
+}
+/* Bottom right text */
+.bottom-right2 {
+  position: absolute;
+  bottom: 27%;
+    right: 37%;
+}
+.div-img {
+  height: 545px;
+  width: 837px;
+  border-radius: 2%;
+}
+
+.playbtn {
+  height: 50px;
+}
+
+.div2 {
+  background-color: #4E4D4D;
+  height: 113px;
+  width: 30%;
+  border-radius: 10px;
+}
+
+.div2-img {
+  height: 113px;
+  width: 100px;
+  border-radius: 10px;
+}
+/* level 3 Bottom left text */
+ .sm-div1{
```

Comment:
Do not need to write separate CSS for each divs, we can reuse same class for each divs. Also it can be done with flexbox, instead of absolute position